doc/src/sgml/Makefile

```diff
@@ -26,10 +26,7 @@ top_builddir = ../../..
 include $(top_builddir)/src/Makefile.global
 
 
-# Don't build the upstream HTML docs or man pages in Greenplum. The Greenplum
-# docs live in the gpdb-doc directory.
```

Comment:
Do we really need to build upstream docs?

Reply:
I returned this place to the 6X.

Reply:
fixed